linux_docker_resources/Dockerfile

```
@@ -33,7 +33,6 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   apt-transport-https
 
 # Add the ROS repositories to the apt sources list.
-RUN if test ${UBUNTU_DISTRO} != jammy; then echo "deb http://repositories.ros.org/ubuntu/testing/ `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.list; fi
 RUN if test ${UBUNTU_DISTRO} = jammy; then echo "deb http://repo.ros2.org/ubuntu/testing/ `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list; fi
```

```suggestion
RUN echo "deb http://repo.ros2.org/ubuntu/testing/ `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list
```

Since any future distros would also be using the ROS 2 repositories I think we can drop the conditional here, which was introduced in #598 specifically because there was no ROS 1 repository for Jammy.

---

Yeah, that's fair.  I kind of waffled on what to do here, and ended up leaving it, but I'll go back and remove it.

---

And fixed in e4bb2b7